homeassistant/components/logbook.py

```diff
@@ -97,7 +116,7 @@ def get(self, request, datetime=None):
             (events.time_fired < end_day))
         events = recorder.execute(query)
 
-        return self.json(humanify(events))
+        return self.json(humanify(events, self.config))
```

Comment:
Let's filter the events before they get passed to humanify, to separate concerns 👍